### ISSUE TITLE
Document Git path dependencies

### ISSUE THREAD

**Issue by nex3** [dart-lang/site-www#314]
In dart-lang/pub#1650, pub adds support for depending on subdirectories of Git repositories. The syntax for this is:

```yaml
dependencies:
  foobar:
    git:
      url: git://github.com/dart-lang/multi_package_repo
      path: pkg/foobar
```

This will depend on the package defined by `pkg/foobar/pubspec.yaml` in the `multi_package_repo` repository.

**Comment by kwalrath** [comment]
@nex3 when will this change be usable by external developers?

**Comment by nex3** [comment]
@kwalrath it will be fully usable in 2.0.0-dev.1, but the first stable release that will support it will be 2.0.0.

**Comment by trentgrover-wf** [comment]
@nex3 is there absolutely zero chance of being able to use this before 2.0?

**Comment by nex3** [comment]
You can use it in dev releases, but there aren't going to be any more stable releases until 2.0.0.

**Comment by dragonfax** [comment]
How can I check what version of flutter I'm using, and how can I opt-in to the dev version. (searched but couldn't find any docs on this).

pubspec.yaml
```
  android_alarm_manager:
    git: git://github.com/flutter/plugins.git
    path: packages/android_alarm_manager
```

flutter packages get
```
Error on line 12, column 5 of pubspec.yaml: A dependency may only have one source.
    git: git://github.com/flutter/plugins.git
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

git status on my flutter install
```
17245-jstillwell:flutter jstillwell$ git status
On branch alpha
Your branch is up-to-date with 'origin/alpha'.
```

**Comment by kwalrath** [comment]
@dragonfax We don't cover (or know anything about) Flutter specifics here. I'd suggest using either Stack Overflow or the Flutter live chat. You can find the links at https://flutter.io/support/.

(This is a good reminder that we need to have more links from www.dartlang.org to flutter.io.)

**Comment by dragonfax** [comment]
Oh shit. I totally put this Issue in on the wrong repo. I have no idea how I did that. Sorry. I was thinking of the flutter repo.

**Comment by dragonfax** [comment]
Sorry. I've had flutter on the brain recently.

**Comment by kwalrath** [comment]
No worries! It's not unreasonable to land on this page and think you might find info relevant to flutter.

**Comment by chalin** [comment]
Closed by #780
